pylab/accounts/middleware.py

```diff
@@ -0,0 +1,14 @@
+from django.middleware.locale import LocaleMiddleware
+from django.utils import translation
+
+
+class UserProfileLocaleMiddleware(LocaleMiddleware):
+    def process_request(self, request):
+        super(UserProfileLocaleMiddleware, self).process_request(request)
+
+        if (hasattr(request.user, 'userprofile') and
```

Comment:
Why you need to check with `hasattr` if it is always guaranteed, that `userprifile` property will be always available? You just need to check if `userprofile` is not `Null`.